src/frontend/static/css/style_resume.css

```diff
@@ -0,0 +1,124 @@
+/* Style for the entire page */
```

Comment:
I will move this file along with all the other css files

Reply:
Okay